source/StartConditionsPanel.cpp

```diff
@@ -0,0 +1,253 @@
+#include "GameData.h"
+#include "GameWindow.h"
+#include "Information.h"
+#include "Interface.h"
+#include "LoadPanel.h"
+#include "MainPanel.h"
+#include "Messages.h"
+#include "PlayerInfo.h"
+#include "Planet.h"
+#include "Preferences.h"
+#include "Rectangle.h"
+#include "Ship.h"
+#include "ShipyardPanel.h"
+#include "StarField.h"
+#include "System.h"
+#include "UI.h"
+#include "WrappedText.h"
+
+#include <SDL2/SDL.h>
+
+#include "gl_header.h"
+
+using namespace std;
+
+namespace 
+{
+    const double LIST_ITEM_SIZE = 20.;
+}
+
+StartConditionsPanel::StartConditionsPanel(PlayerInfo &player, UI &gamePanels, LoadPanel *loadPanel)
+    : player(player), gamePanels(gamePanels), loadPanel(loadPanel)
+{
+    if(GameData::Start().size())
+    {
+        chosenStart = GameData::Start()[GameData::Start().size()-1];    
+    }
+    else
+    {
+        chosenStart = nullptr;
+    }
+    
+}
+
+
+void StartConditionsPanel::Draw()
+{
+    glClear(GL_COLOR_BUFFER_BIT);
+
+    Information info;
+
+    string descriptionText; // String that will be shown in the description panel
+
+    if(chosenStart){
+        info.SetCondition("chosen start");
+        if(chosenStart->GetSprite())
+            info.SetSprite("start sprite", chosenStart->GetSprite());
+        info.SetString("name", chosenStart->GetName());
+        info.SetString("description", chosenStart->GetDescription());
+        info.SetString("planet", chosenStart->GetPlanet()->Name());
+        info.SetString("system", chosenStart->GetSystem()->Name());
+        info.SetString("date", chosenStart->GetDate().ToString());
+        info.SetString("credits", to_string(chosenStart->GetAccounts().Credits()));
+
+        descriptionText = chosenStart->GetDescription();
+    }
+    else if (!GameData::Start().size())
+    {
+        descriptionText = "No start scenarios were defined!\n\nMake sure that you installed Endless Sky and all of your plugins properly";
+    }
+    else
+    {
+        descriptionText = "";        
+    }
+
+    GameData::Background().Draw(Point(), Point());
+    GameData::Interfaces().Get("menu background")->Draw(info, this);
+    GameData::Interfaces().Get("start conditions menu")->Draw(info, this);
+    GameData::Interfaces().Get("menu start info")->Draw(info, this);
+    
+    const Font &font = FontSet::Get(14);
+
+    WrappedText *text = new WrappedText(font);
+
+    text->SetAlignment(WrappedText::LEFT);
+    text->SetWrapWidth(210);
+    text->Wrap(descriptionText);
+
+    // Only allow draws inside the description box
+    glScissor(
+        -105 + GameWindow::TrueWidth() / 2,
+        -160 + GameWindow::TrueHeight() / 2,
+        210,
+        320);
+    glEnable(GL_SCISSOR_TEST);
+    text->Draw(Point(-105, -160+descriptionScroll), Color(1.,1.,1.));
+    glDisable(GL_SCISSOR_TEST);
+
+    Point point(-470., -157. + listScroll);
+    for(const auto &it : GameData::Start())
+    {
+        Rectangle zone(point + Point(110., 7.), Point(230., LIST_ITEM_SIZE));
```

Comment:
Don't use an imperative draw code. The user should be able to supply their interface definition and sprite, and this panel's draw code needs to still work correctly. The rest of the game isn't quite at that point, but I don't want to add _any_ new panels that will add to the existing tech debt.